### PR TITLE
Fix singular assure

### PR DESCRIPTION
**Pull request by tthsqe12**

### CODE REVIEW

File: src/Rings/mpoly-ideals.jl [hunk 1]
```diff
@@ -162,10 +162,21 @@ ideal(x^3*y - x*y - y^3, x^4 - x^2 - x*y^2)
...
+  end
+  o = default_ordering(base_ring(I))
+  singular_assure(I, o)
   si = I.gens.S
   for J in Js
-    singular_assure(J)
+    singular_assure(J, o)
     si = Singular.intersection(si, J.gens.S)
   end
   return MPolyIdeal(base_ring(I), si)
```
Comment (tthsqe12) [inline src/Rings/mpoly-ideals.jl:182]: @ederc @YueRen is this how you would deal with incompatible orderings in singular operations?

Comment (YueRen) [reply src/Rings/mpoly-ideals.jl:182]: I’m not 100% familiar with the functions used, but it looks sensible to me. Can the default_ordering be set manually? If yes that would be great.

Comment (tthsqe12) [reply src/Rings/mpoly-ideals.jl:182]: sure, via the ordering keyword we can do that.

Comment (ederc) [reply src/Rings/mpoly-ideals.jl:182]: Looks good to me.

**Pull request closed without merging**